lib/widgets/todo_item.dart

```diff
@@ -1,52 +1,42 @@
-
-import 'dart:developer';
-
 import 'package:flutter/material.dart';
 import 'package:todo_app/constants/colors.dart';
 import 'package:todo_app/widgets/alert_builder.dart';
+import '../models/to_do.dart';
 
 class ToDoItem extends StatelessWidget {
-  const ToDoItem({Key? key}) : super(key: key);
+  final Todo todo;
+  final Function onTodoChanged;
+
+  const ToDoItem({Key? key, required this.todo, required this.onTodoChanged})
+      : super(key: key);
```

Comment:
Key? key, : super(key: key); kan ersättas med super.key:

` const ToDoItem({super.key, required this.todo, required this.onTodoChanged})`